### PR TITLE
implemented task and tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+looking and behaving just as described below.
+
+Please **do not** modify or replace the HTML itself. Do take a look at what’s 
+inside `index.html` though.
+
+The page consists of a header, a footer, and 9 blocks between them.
+
+The width of the blocks must be 300 pixels. The height of each of the blocks 
+(including both header and footer) has to be 100 pixels. When hovered (the 
+cursor is on the block), it must increase to 200 pixels.
```
Comment (roman-mate) [inline readme.md]:
```suggestion
cursor is on the block), height must increase to 200 pixels.
```

File: readme.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+The color of the header must be blue, the color of the footer must be black, and 
+the colors of the blocks in between must be red, then green, then orange, then
```
Comment (roman-mate) [inline readme.md]:
```suggestion
the colors of the blocks in between must be the following
1. red
1. green
2. orange
... 
```

File: readme.md [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+The width of the blocks must be 300 pixels. The height of each of the blocks 
+(including both header and footer) has to be 100 pixels. When hovered (the 
+cursor is on the block), it must increase to 200 pixels.
+The color of the header must be blue, the color of the footer must be black, and 
+the colors of the blocks in between must be red, then green, then orange, then 
+again red, then green, then orange, and once again red, then green, then orange.
+
+There should be no vertical spacing between the blocks and the edges of the 
+page. Place the blocks so that they are horizontally centered on the page (you 
+can use `margin: auto` for that).
```
Comment (roman-mate) [inline readme.md]: I think we are providing too many details. They should figure out themselves how to center the element.

Comment (gestap) [reply readme.md]: Agree with Roman.

File: readme.md [hunk 4]
```diff
@@ -1,15 +1,31 @@
...
+(including both header and footer) has to be 100 pixels. When hovered (the 
+cursor is on the block), height must increase to 200 pixels.
+The color of the header must be blue, the color of the footer must be black, and 
+the colors of the blocks in between must be the following
+1. red
+1. green
+2. orange
+
+... 
+again red, then green, then orange, and once again red, then green, then orange.
```
Comment (roman-mate) [inline readme.md]: This can be removed now. And let's remove `...` above.

File: readme.md [hunk 5]
```diff
@@ -1,15 +1,31 @@
...
+The page consists of a header, a footer, and 9 blocks between them.
+
+The width of the blocks must be 300 pixels. The height of each of the blocks 
+(including both header and footer) has to be 100 pixels. When hovered (the 
+cursor is on the block), height must increase to 200 pixels.
+The color of the header must be blue, the color of the footer must be black, and 
+the colors of the blocks in between must be the following
+1. red
+1. green
+2. orange
```
Comment (roman-mate) [inline readme.md]:
```suggestion
1. orange
```

**Pull request merged**